### PR TITLE
AttachEntityAttributes action

### PR DESCRIPTION
**Pull request by grooze**

### CODE REVIEW

File: appmetr/src/main/java/com/appmetr/android/AppMetr.java [hunk 1]
```diff
@@ -385,6 +386,33 @@ public static boolean verifyPayment(String purchaseInfo, String signature, Strin
...
+     * @param properties - attributes to attach
+     */
+    public static void attachEntityAttributes(String entityName, String entityValue, JSONObject properties) {
+        try {
+            if(TextUtils.isEmpty(entityName) || TextUtils.isEmpty(entityValue)) {
+                Log.e(TAG, "AttachEntityAttributes failed: entity name and value can't be empty");
+                return;
+            }
+            
+            if (properties == null) {
```
Comment (chemist777) [inline appmetr/src/main/java/com/appmetr/android/AppMetr.java]: should not be empty

Comment (grooze) [reply appmetr/src/main/java/com/appmetr/android/AppMetr.java]: what to do? log exception and exit?

**Comment by chemist777** [comment]
Throw illegalargumentexception

File: appmetr/src/main/java/com/appmetr/android/AppMetr.java (hunk 1 above)
Comment (chemist777) [reply appmetr/src/main/java/com/appmetr/android/AppMetr.java]: Throw illegalargumentexception

Comment (grooze) [reply appmetr/src/main/java/com/appmetr/android/AppMetr.java]: Done

**Pull request merged**